.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```

Comment:
- [ ] As requested by previous reviewers you should put the  `test.md file` in `.gitignore` file and It should not present on your repo. Please fix it as I see the test.md file in your repo.

Reply:
@Donmark2k, Please try to push the code and check whether there is test.md file present on the repo or not.

Reply:
@Donmark2k, normally it happens when you already upload files, and after 
  that you add your .gitignore file.